ecommerce/ecommerce/urls.py

```diff
@@ -16,11 +16,29 @@
 from django.contrib import admin
 from django.urls import path, include
 
+from baskets.views import BasketViewSet, BasketItemViewSet
+from customers.views import CustomerViewSet, AddressViewSet, CityViewSet, CountryViewSet
 from ecommerce.router import router
+from orders.views import OrderViewSet, OrderItemViewSet, BillingAddressViewSet, ShippingAddressViewSet, \
+    OrderBankAccountViewSet
+from payments.views import BankViewSet, BankAccountViewSet
 from products.views import ProductViewSet, CategoryViewSet
 
 router.register("products", ProductViewSet)
 router.register("categories", CategoryViewSet)
+router.register("banks", BankViewSet)
+router.register("bank accounts", BankAccountViewSet)
```

Comment:
genel convention'a uygun olarak endpoint isimlerinde boşluk bırakmak yerine "-" kullanmak iyi olabilir. Örneğin; bank-accounts

Reply:
https://restfulapi.net/resource-naming/ 
burdan daha ayrıntlı bakabilirsin

Reply:
Teşekkür ederim